tutoraspects/templates/openedx-assets/queries/course_tag_filter.sql

```diff
@@ -0,0 +1,10 @@
+{% raw -%}
+{% if filter_values("tag") != [] %}
```

Comment:
You can add another check to verify if the filter exists, and add it to the standard filters